Gemfile

```diff
@@ -8,10 +8,15 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
   gem "simplecov", require: false
+  # Override the coffee-rails version in engine_cart for rails 4.2
+  case ENV['RAILS_VERSION']
+  when /^4.2/
+    gem 'coffee-rails', '~> 4.2.0'
```

Comment:
This seems like it conflicts with line 44.  Can you say more about why we need both?